package.json

```diff
@@ -10,6 +10,7 @@
     "test": "npm run unit && npm run integration && npm run versioned",
     "unit": "tap --test-regex='(\\/|^tests\\/unit\\/.*\\.test\\.js)$' --no-coverage",
     "versioned": "npm run versioned:npm6",
+    "versioned:folder": "versioned-tests --minor -i 2",
```

Comment:
This just made it easier to run only the versioned tests I wanted locally via `npm run versioned:folder ./tests/versioned/apollo-federation`